content/en/guides/models/track/runs/_index.md

```diff
@@ -348,26 +348,30 @@ https://wandb.ai/<team-name>/<project-name>/runs/<run-id>
 Where values enclosed in angle brackets (`< >`) are placeholders for the actual values of the team name, project name, and run ID.
 
 ### Customize how runs are displayed
-This section shows how to customize how runs are displayed in your project's **Workspace** and **Runs** tab, which share the same display configuration.
+This section shows how to customize how runs are displayed in your project's Workspace and Runs table.
 
 {{% alert %}}
 A workspace is limited to displaying a maximum of 1000 runs, regardless of its configuration.
 {{% /alert %}}
 
+#### Add or remove columns
 
-To customize which columns are visible:
-1. In the project sidebar, navigate to the **Runs** tab.
+To customize which columns are visible in the Runs table or Workspace:
+1. In the project sidebar, navigate to the **Runs** tab or the **Workspace** tab.
 1. Above the list of runs, click **Columns**.
 1. Click the name of a hidden column to show it. Click the name of a visible column to hide it.
-  
     You can optionally search by column name using fuzzy search, an exact match, or regular expressions. Drag columns to change their order.
 1. Click **Done** to close the column browser.
 
+#### Sort runs by column
+
 To sort the list of runs by any visible column:
 
 1. Hover over the column name, then click its action `...` menu.
 1. Click **Sort ascending** or **Sort descending**.
 
+#### Pin columns
+
 Pinned columns are shown on the right-hand side. Unpinned columns are shown on the left-hand side of the **Runs** tab and are not shown on the **Workspace** tab.
```

Comment:
I think this might be backward and I think I am the one that introduced this sentence. 😭  Confirmed in the UI.
```suggestion
Pinned columns are shown on the left-hand side. Unpinned columns are shown on the right-hand side of the **Runs** tab and are not shown on the **Workspace** tab.
```